index.html

```diff
@@ -0,0 +1,141 @@
+                <div class="d-flex flex-row align-items-center pb-2">
+                    <a href="#" class="text-white font-weight-bolder">LOGIN</a>
+                    <button class="subscribe border-0" >SUBSCRIBE ></button>
+                </div>
+            </section>
+            <section class="nav2 bg-white overflow-hidden position-relative py-2 d-lg-block d-none">
+                <ul class="d-flex align-items-center justify-content-between list-unstyled">
+                    <li><a href="#">U.S.</a></li>
+                    <li><a href="#">World</a></li>
+                    <li><a href="#">Business</a></li>
+                    <li><a href="#">Tech & Science</a></li>
+                    <li><a href="#">Culture</a></li>
+                    <li><a href="#">Newsgeek</a></li>
+                    <li><a href="#">Sports</a></li>
+                    <li><a href="#">Health</a></li>
+                    <li><a href="#">The Debate</a></li>
+                    <li><a href="#">Experts</a></li>
+                    <li><a href="#">Vantage</a></li>
+                    <li class="search-btn d-flex flex-row align-items-center">
+                        <input type="search" placeholder="Search" class="border-0">
+                        <i class="fas fa-search" aria-hidden="true"></i>
+                    </li>
+                </ul>
+            </section>
+            <section class="nav-3 bg-white overflow-hidden position-relative pt-0 pb-2 d-lg-block d-none">
+                <ul class="list-unstyled d-flex flex-row font-weight-bold ">
+                    <li>TRENDING</li>
+                    <li><a href="#">JOE BIDEN</a></li>
+                    <li><a href="#">DONALD TRUMP</a></li>
+                    <li><a href="#">INAUGURATION</a></li>
+                    <li><a href="#">CORONA VIRUS</a></li>
+                    <li><a href="#">STIMULUS PACKAGE</a></li>
+                    <li><a href="#">VACCINE</a></li>
+                    <li><a href="#">CAPITOL RIOTS</a></li>                                                                                                                                                                                                                                                       
+                </ul>
+            </section>
```

Comment:
- [ ] This section is missing the borders above and below it. Additionally, the spacing could be improved. See the pictures below for a closer look
Your clone 👇🏾 
<img width="1432" alt="Screen Shot 2021-01-25 at 7 58 14 PM" src="https://user-images.githubusercontent.com/48804258/105752507-ab86ff80-5f47-11eb-86cb-4bc0a8440c25.png">

The original page 👇🏾 
<img width="1317" alt="Screen Shot 2021-01-25 at 7 59 02 PM" src="https://user-images.githubusercontent.com/48804258/105752594-c78aa100-5f47-11eb-98af-a7bc6a89b7b3.png">